xdsl/transforms/lower_riscv_func.py

```diff
@@ -59,9 +59,80 @@ def match_and_rewrite(self, op: riscv_func.SyscallOp, rewriter: PatternRewriter)
         rewriter.replace_matched_op(ops, new_results=new_results)
 
 
+SCALL_EXIT = 93
+"""
+93 is the number of the `exit` syscall on RISCV.
```

Comment:
Nit: Not "on RISCV," but rather on our first target (riscemu). This is very much API/ABI specific.

Reply:
Also, I don't think it makes sense to have a concept of specific syscall codes at this level of abstraction, right? The higher-level program (or layer) takes care of selecting the right syscall code.

Reply:
Yep, this is unused now, I think, should be removed